projects/kit/components/input-date/multi-date/input-date-multi.directive.ts

```diff
@@ -0,0 +1,42 @@
+import {Directive} from '@angular/core';
+import {TuiDay} from '@taiga-ui/cdk';
+import {AbstractTuiTextfieldHost, tuiAsTextfieldHost} from '@taiga-ui/core';
+
+import {tuiImmutableUpdateInputDateMulti} from './immutable-update';
+import {TuiInputDateMultiComponent} from './input-date-multi.component';
+
+@Directive({
+    selector: 'tui-input-date[multiple]',
+    providers: [tuiAsTextfieldHost(TuiInputDateMultiDirective)],
+})
+export class TuiInputDateMultiDirective extends AbstractTuiTextfieldHost<TuiInputDateMultiComponent> {
+    override get value(): string {
+        return String(this.host.value);
```

Comment:
1) This is the default behavior, no need to override it.
2) I have a feeling this is wrong, in case of multi date we do not need an entire array stringified for value, also it might be the same mistake in multi select and input tag – this `value` is for nested `input tuiTextfield`, we do not need an entire stringified array inside it.

Reply:
what do you mean I should do?